operate/cli.py

```diff
@@ -946,15 +946,14 @@ def _daemon(
 @_operate.command(name="quickstart")
 def qs_start(
     config: Annotated[str, params.String(help="Quickstart config file path")],
-    attended: Annotated[
-        str, params.String(help="Run in attended/unattended mode (default: true")
-    ] = "true",
+    attended: Annotated[str, params.String(help="Run in attended/unattended mode (default: true")] = "true",
+    build_only: Annotated[bool, params.Boolean(help="Only build the service without running it")] = False,
```

Comment:
can you ensure this is green
```
tox -p -e bandit -e safety -e black -e isort -e flake8 -e mypy -e pylint
```
I have already fixed everything on the `feat/multi-agent-middleware` branch, just merge that into your branch if you find a lot of issues. Otherwise just fix yours.

Reply:
update: `feat/multi-agent-middleware` is merged to `staging` now. Please re-target it to `staging`.